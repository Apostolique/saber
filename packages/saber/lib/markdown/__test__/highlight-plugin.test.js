const Markdown = require('saber-markdown')
const fenceOptionsPlugin = require('../highlight-plugin')
const createEnv = require('./create-env')

test('main', () => {
  const md = new Markdown()
  const { env } = createEnv()
  md.use(fenceOptionsPlugin)
  const html = md.render(
    `
\`\`\`vue
<div>hehe</div>
\`\`\`
  `,
    env
  )
  expect(html).toMatchSnapshot()
})
