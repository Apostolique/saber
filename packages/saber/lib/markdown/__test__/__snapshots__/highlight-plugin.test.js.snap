// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`main 1`] = `"<div class=\\"saber-highlight\\" v-pre=\\"\\" data-lang=\\"vue\\"><pre class=\\"saber-highlight-code language-vue\\"><code class=\\"language-vue\\">&lt;div&gt;hehe&lt;/div&gt;</code></pre></div>"`;
